packages/react/src/control-base/ControlBase.tsx

```diff
@@ -44,7 +43,11 @@ export const ControlBase = forwardRef<HTMLDivElement, ControlBaseProps>(
         </Text>
 
         {endDecorator && (
-          <Box asChild {...styles.decorator()}>
+          <Box
+            asChild
+            color={props.disabled ? "fg.disabled" : "fg.secondary"}
```

Comment:
Let's add the fontSize here as well

In majority cases I'm assuming we'll need small text